axis_backend/apps/authentication/models/user_client.py

```
@@ -0,0 +1,77 @@
+            models.Index(fields=['client']),
+            models.Index(fields=['deleted_at']),
+        ]
+        constraints = [
+            models.UniqueConstraint(
+                fields=['user', 'client'],
+                condition=models.Q(deleted_at__isnull=True),
+                name='unique_user_client_active'
+            )
+        ]
+
+    def __str__(self):
```

Ruff (0.14.5)

Missing return type annotation for special method `__str__`

Add return type annotation: `str`

(ANN204)